packages/cli/tests/e2e/frontend/CanScaffoldDotNetProject.tests.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @author Yefu Wang <yefuwang@microsoft.com>
+ */
+
+import { describe } from "mocha";
+import * as chai from "chai";
+import fs from "fs-extra";
+import path from "path";
+import { environmentManager } from "@microsoft/teamsfx-core";
+import { getSubscriptionId, getTestFolder, getUniqueAppName, cleanUp } from "../commonUtils";
+import AppStudioLogin from "../../../src/commonlib/appStudioLogin";
+import { CliHelper } from "../../commonlib/cliHelper";
+import { Capability } from "../../commonlib/constants";
+import mockedEnv, { RestoreFn } from "mocked-env";
+
+describe(".NET projects", function () {
+  const testFolder = getTestFolder();
+  const appName = getUniqueAppName();
+  const subscription = getSubscriptionId();
+  const projectPath = path.resolve(testFolder, appName);
+  const env = environmentManager.getDefaultEnvName();
+
+  after(async () => {
+    // clean up
+    await cleanUp(appName, projectPath, false, false, false);
+  });
+  describe("teamsfx new a .NET project", async function () {
+    const envs = [{ TEAMSFX_CLI_DOTNET: "true" }];
```

Comment:
When are we going to deprecate this feature flag?

Reply:
Scaffolding a .NET project is not in VS's ga scope. It can be removed when the feature is ready for public consumption. For now, it's now an internal feature for E2E testing.